data/json/items/gun/pebble.json

```diff
@@ -59,7 +59,7 @@
     "weight": 254,
     "volume": 1,
     "price": 500,
-    "bashing": 0,
+    "bashing": 1,
```

Comment:
If it's `bashing 1` it might as well be `bashing 0` and therefore dropped entirely